src/memote/suite/tests/test_consistency.py

```diff
@@ -153,11 +154,16 @@ def test_detect_energy_generating_cycles(model, met):
 
     """
     ann = test_detect_energy_generating_cycles.annotation
-    if met not in model.metabolites:
+    # Test if the metabolite is present in the model.
+    main_comp = helpers.find_compartment_id_in_model(model, "c")
+    try:
+        helpers.find_met_in_model(model, met, main_comp)[0]
+    except:
```

Comment:
It's a good idea to be more explicit about the expected exception.

```suggestion
    except ValueError:
        pytest.skip("Metabolite {} is not in the MetaNetX shortlist.".format(met))
    except (RuntimeError, IndexError):
```

Reply:
I know bare except statements are against the style guide, but specifying `ValueError` doesn't actually skip the tests where the metabolite isn't present.

Here is a screenshot from my report when L161 was `except ValueError:`:
![image](https://github.com/opencobra/memote/assets/40765908/74628b97-c13a-4b83-b87b-f04be0044f97)

And here is a screenshot for when L161 was just `except:`:
![image](https://github.com/opencobra/memote/assets/40765908/c619b91a-ac15-4e3e-89bc-627f55e828a6)

Reply:
Nevermind, you were right, I copied your code incorrectly!